### PR TITLE
#555: `nimble test` removes the created binaries

### PR DESCRIPTION
**Pull request by jabbalaci**

### CODE REVIEW

File: src/nimble.nim [hunk 1]
```diff
@@ -1002,7 +1002,17 @@ proc test(options: Options) =
       optsCopy.action.compileOptions = @[]
       optsCopy.action.compileOptions.add("-r")
       optsCopy.action.compileOptions.add("--path:.")
+      let
+        binFileName = joinPath(dir, name) & (if defined(windows): ".exe" else: "")
```
Comment (dom96) [inline src/nimble.nim]: This can be simplified to:

```
binFileName = file.path.changeFileExt(ExeExt)
```

Comment (dom96) [reply src/nimble.nim]: btw, use ``/`` instead of ``joinPath`` for future reference.

**Comment by jabbalaci** [comment]
I pushed another commit and changed that line. However, I might have messed up the "resolve conversation".

**Comment by dom96** [comment]
Not sure what you mean, but looks good.

**Pull request merged**